lib/landrush/util/dnsmasq.rb

```diff
@@ -0,0 +1,10 @@
+module Landrush
+  module Util
+    class Dnsmasq
+      def self.nm_managed?
+        nm_config = Pathname('/etc/NetworkManager/NetworkManager.conf')
+        File.exist?(nm_config) && File.readlines(nm_config).grep(/^dns=dnsmasq$/).any?
```

Comment:
I am using fedora 24 and when I grep dns=dnsmasq it doesn't have that string by default. I do have dnsmasq service up and running.
 
```
# cat /etc/NetworkManager/NetworkManager.conf 
# Configuration file for NetworkManager.
#
# See "man 5 NetworkManager.conf" for details.
#
# The directory /usr/lib/NetworkManager/conf.d/ can contain additional configuration
# snippets installed by packages. These files are read before NetworkManager.conf
# and have thus lowest priority.
# The directory /etc/NetworkManager/conf.d/ can contain additional configuration
# snippets. Those snippets override the settings from this main file.
#
# The files within one conf.d/ directory are read in asciibetical order.
#
# If /etc/NetworkManager/conf.d/ contains a file with the same name as
# /usr/lib/NetworkManager/conf.d/, the latter file is shadowed and thus ignored.
# Hence, to disable loading a file from /usr/lib/NetworkManager/conf.d/ you can
# put an empty file with the same name.
#
# If two files define the same key, the one that is read afterwards will overwrite
# the previous one.

[main]
#plugins=ifcfg-rh,ibft


[logging]
# When debugging NetworkManager, enabling debug logging is of great help.
#
# Logfiles contain no passwords and little sensitive information. But please
# check before posting the file online. You can also personally hand over the
# logfile to a NM developer to treat it confidential. Meet us on #nm on freenode.
# Please post full logfiles except minimal modifications of private data.
#
# You can also change the log-level at runtime via
#   $ nmcli general logging level TRACE domains ALL
# However, usually it's cleaner to enable debug logging
# in the configuration and restart NetworkManager so that
# debug logging is enabled from the start.
#
# You will find the logfiles in syslog, for example via
#   $ journalctl -u NetworkManager
#
#level=TRACE
#domains=ALL

# cat /etc/NetworkManager/NetworkManager.conf  | grep dnsmasq

$ systemctl status dnsmasq.service 
● dnsmasq.service - DNS caching server.
   Loaded: loaded (/usr/lib/systemd/system/dnsmasq.service; disabled; vendor preset: disabled)
   Active: active (running) since Thu 2016-11-03 16:25:32 IST; 21min ago
 Main PID: 8099 (dnsmasq)
    Tasks: 1 (limit: 512)
   CGroup: /system.slice/dnsmasq.service
           └─8099 /usr/sbin/dnsmasq -k
```